release/start-acs/action.yml

```diff
@@ -49,6 +49,7 @@ runs:
         KUBECONFIG: ${{ inputs.kubeconfig }}
         STACKROX_DIR: ${{ inputs.stackrox-dir }}
         NAME: ${{ inputs.name }}
+        ROX_TELEMETRY_STORAGE_KEY_V1: R5fMyO9n0gibSGzOXtlP2qCFWCGb8uoW
```

Comment:
@0x656b694d this need not to be a secret?

Reply:
Not really.